README.md

```diff
@@ -1,3 +1,19 @@
+## About the Project
+
+The TestScript Execution Engine is to support FHIR testing in an integrated and systematic way by providing the following features.
+
+* A general purpose test execution tool to be implemented in a variety of use cases
+* Support TestScript / TestReport Resource types as inputs and outputs to be aligned with FHIR architecture
+* (TBA) Support an end-to-end pipeline of FHIR testing including authoring exception handling and analytics
```

Comment:
I'll defer to @jawalonoski on whether or not we want to advertise this intention

Reply:
Sure, I believe there will be rooms for polishing and refining the introduction as to where the project scope and direction go.